### DISCUSSION TITLE
Break packages into own repositories

### DISCUSSION THREAD

**Discussion by dburles** [open-wc/open-wc#2715] [Ideas]
The monorepo is a hinderance to contributing. I would love to make a contribution by refactoring `import-maps-resolve` to ESM so that it can also run on Deno. However, in order to do so, I must first install **66** dependencies, which installs **1972** modules (**_915MB_**!).

**Comment by Westbrook** [comment]
The mono-repo is an important part of our being able to maintain the various interrelated projects that we ship as a community, so we’ll not likely be doing any major deconstruction of the code in the foreseeable future.
